app/src/main/java/org/simple/clinic/summary/PatientSummaryScreenController.kt

```diff
@@ -157,7 +157,7 @@ class PatientSummaryScreenController @AssistedInject constructor(
         .ofType<PatientSummaryItemChanged>()
```

Comment:
Rename this method to `populateDrugs`

Reply:
I intentionally did not name this because this method will be removed in #1038.